Assets/Scripts/Game/UserInterfaceWindows/HardStrings.cs

```diff
@@ -251,6 +251,66 @@ public class HardStrings
         public const string cannotTravelWithEnemiesNearby = "You cannot travel with enemies nearby.";
 
         public const string powersUnknown = "Powers unknown.";
+
+        public static readonly string[] itemPowers = { "Cast when used:",
```

Comment:
At some point, I need to help with a better way to store this text outside of HardStrings. My text tables are only a part-measure for now and have a few limitations (e.g. one text result per key, commas break table schema in text). This is fine here for now though, it can be moved to a new text setup later once that exists.